docs/guide/core-widgets.md

```diff
@@ -142,7 +167,31 @@ module.exports = {
   }
 }
 ```
-
+### Using Markdown in the Rich Text Widget
+The rich text widget has limited support for using [Markdown syntax](https://www.markdownguide.org/cheat-sheet/). By default, a single octothorpe ('#') followed by a space will convert the following text into an H1 heading. Several other Markdown shortcuts are enabled by default, including double tilde (`~~ strike-though~~`) for strike-through and double asterisk (`**bold**`) for bold. Support for additional heading shortcuts, e.g. `## `H2 heading` requires that you define them in your styles section. So for the default settings, headings `# H1`, `## H2`, `### H3`, and `#### H4` are defined.
+
+
+| Element | Syntax | Enabling |
+| --- | --- | --- |
+| `h1` | `# heading 1` | Default |
```

Comment:
Are you saying H1 works even if there's no such thing in `styles`? Because even if it lets you type it, it won't stick on the server end. Hopefully this doesn't work at all if it's not in `styles`, if not we should address it as a bug rather than documenting it as a feature.

Reply:
I modified this to describe the actual behavior. If you think it is still too confusing, I can turf the Markdown portion for now and open a ticket. Several people between Productboard and Discord have asked about this feature.